src/vs/workbench/services/textfile/common/textEditorService.ts

```diff
@@ -266,4 +266,4 @@ export class TextEditorService extends Disposable implements ITextEditorService
 	}
 }
 
-registerSingleton(ITextEditorService, TextEditorService, false /* do not change: https://github.com/microsoft/vscode/issues/137675 */);
+registerSingleton(ITextEditorService, TextEditorService, false);
```

Comment:
Not worth keeping the comment?

Reply:
@alexr00 oh thanks, restored it. I had thought it was for an issue I recently fixed.